--- origsrc/totem-pl-parser-2.32.2/plparse/totem-pl-parser.c	2011-01-19 09:57:57.000000000 -0600
+++ src/totem-pl-parser-2.32.2/plparse/totem-pl-parser.c	2011-01-30 17:49:57.066415100 -0600
@@ -2477,7 +2477,11 @@ totem_pl_parser_can_parse_from_filename
 gboolean
 totem_pl_parser_can_parse_from_uri (const char *uri, gboolean debug)
 {
+#ifndef TOTEM_PL_PARSER_MINI
 	return totem_pl_parser_is_videosite (uri, debug);
+#else
+	return FALSE;
+#endif /* TOTEM_PL_PARSER_MINI */
 }
 
 #ifndef TOTEM_PL_PARSER_MINI
