--- origsrc/totem-pl-parser-2.32.2/configure.in	2011-01-31 18:57:20.871747800 -0600
+++ src/totem-pl-parser-2.32.2/configure.in	2011-01-31 18:58:56.437514400 -0600
@@ -128,7 +128,7 @@ PKG_CHECK_MODULES(TOTEM_PLPARSER, [$pkg_
 AC_SUBST(TOTEM_PLPARSER_CFLAGS)
 AC_SUBST(TOTEM_PLPARSER_LIBS)
 
-PKG_CHECK_MODULES([TOTEM_PLPARSER_MINI], [gio-2.0])
+PKG_CHECK_MODULES([TOTEM_PLPARSER_MINI], [gio-2.0 $QUVI])
 
 GNOME_COMMON_INIT
 GNOME_DEBUG_CHECK
--- origsrc/totem-pl-parser-2.32.2/plparse/Makefile.am	2010-12-14 08:01:56.000000000 -0600
+++ src/totem-pl-parser-2.32.2/plparse/Makefile.am	2011-01-31 18:59:29.883972700 -0600
@@ -106,6 +106,8 @@ libtotem_plparser_mini_la_SOURCES =		\
 	totem-pl-parser-qt.h			\
 	totem-pl-parser-smil.c			\
 	totem-pl-parser-smil.h			\
+	totem-pl-parser-videosite.c		\
+	totem-pl-parser-videosite.h		\
 	totem-pl-parser-wm.c			\
 	totem-pl-parser-wm.h			\
 	totem-pl-parser-xspf.c			\
