--- origsrc/totem-pl-parser-3.4.1/plparse/plparser.symbols	2010-12-14 17:39:51.000000000 -0600
+++ src/totem-pl-parser-3.4.1/plparse/plparser.symbols	2012-05-04 18:55:35.246984100 -0500
@@ -14,7 +14,6 @@ totem_pl_parser_can_parse_from_uri
 totem_pl_parser_error_get_type
 totem_pl_parser_error_quark
 totem_pl_parser_get_type
-totemplparser_marshal_VOID__STRING_STRING_STRING
 totem_pl_parser_new
 totem_pl_parser_parse
 totem_pl_parser_parse_async
