--- origsrc/totem-pl-parser-3.4.1/configure.in	2012-04-16 09:57:18.000000000 -0500
+++ src/totem-pl-parser-3.4.1/configure.in	2012-05-04 19:03:23.662776000 -0500
@@ -75,13 +75,13 @@ AC_ARG_ENABLE(gmime-i-know-what-im-doing
 
 if test "x$enable_gmime" = "xyes" ; then
    PKG_CHECK_MODULES(GMIME,  
-		     gmime-2.4,
-                     [have_gmime=yes GMIME=gmime-2.4], 
+		     gmime-2.6,
+                     [have_gmime=yes GMIME=gmime-2.6], 
                      [have_gmime=no])
    if test "x$have_gmime" != "xyes" ; then
      PKG_CHECK_MODULES(GMIME,  
-		       gmime-2.6,
-		       [have_gmime=yes GMIME=gmime-2.6], 
+		       gmime-2.4,
+		       [have_gmime=yes GMIME=gmime-2.4], 
 		       [have_gmime=no])
    fi
 
